### ISSUE TITLE
Merge content from the MIMIC Cookbook repository

### ISSUE THREAD

**Issue by tompollard** [MIT-LCP/mimic-code#46]
There is an oldish MIMIC Cookbook repository at:
https://github.com/MIT-LCP/mimic-cookbook

We should 
1. merge the content if it hasn't been merged already
2. delete the cookbook repository.

**Comment by alistairewj** [comment]
Let's tick this off one at a time and get rid of the repository. A lot has already been done by @erinhong - here's a list:

* [ ] Age (age.sql)
* [ ] Basic Patient Info (basic_patient_info.sql)
* [ ] Billirubun (bun.sql)
* [ ] SAPS-I formula (create_saps_formula.sql)
* [ ] Calculate Elixhauser scores (exlihauser_comorbidity.sql)
* [ ] Glasgow Coma Score (gcs.sql)
* [ ] Blood glucose (glucose.sql)
* [ ] Bicarbonate (hco.sql)
* [ ] Hematocrit (hct.sql)
* [ ] Heart Rate (heart_rate.sql)
* [ ] Height (height.sql)
* [ ] Insulin delivered (insulin_doses.sql)
* [ ] Blood pressure from hospital survivors (min_surviving_bp.sql)
* [ ] Number of patients (number_of_patients.sql)
* [ ] Blood serum potassium (potassium.sql)
* [ ] Respitation rate (rr.sql)
* [ ] Create SAPS-I (saps_create_24hr_minmax.sql)
* [ ] Systolic Blood Pressure (sbp.sql)
* [ ] Sodium in blood serum (sodium.sql)
* [ ] Insert SOFA scores (sofa_score_inserts.sql)
* [ ] Urine Output (uo.sql)
* [ ] White blood cell count (wbc.sql)

In addition there is some antimicrobial code which we could add:

* [ ] Antimicrobial_Groupings.txt
* [ ] BroadSpectrum IV Antimicrobials.sql

**Comment by MissClarinha** [comment]
Hi, I'm new here and I have a question about the uo.sql file. I have version v1.4 of mimic-iii so the table ioevents doesn't exist anymore, so I changed it to outputevents, but it gives me an error on "volume".
Furthmore, why was it choosen those "itemid"? And what about itemid from other tables like chart-, microbiology- and procedure- events?

**Comment by alistairewj** [comment]
Hello, welcome. Thanks for bringing this up. The cookbook was a bit out of date. The column name was changed to VALUE for consistency with other tables. I've updated all the code, including uo.sql, and it should all work correctly now.

Finding `ITEMID` values involves searching the d_items table `LABEL` column for the concept you are interested in. The `ITEMID` in the queries were manually determined by us.
